public/assets/elfload.js

```diff
@@ -103,6 +103,7 @@ function loadElf(binfile, filename, filesList) {
 
     // start running program
     RISCV.pc = elf["e_entry"].getLowBits();
+    //console.log("here");
```

Comment:
if its not too inconvenient, could we get rid of debug prints in code before merging?